src/plugins/intel_npu/src/plugin/npuw/partitioning/online/compiler.cpp

```diff
@@ -134,6 +134,9 @@ class Compiler {
         for (const auto& isol : ov::npuw::online::util::getIsolates(ov::npuw::online::util::ISOL_PRESETS.at("FAKE"))) {
             isolates.push_back(isol);
         }
+        for (const auto& isol : ov::npuw::online::util::getIsolates(ov::npuw::online::util::ISOL_PRESETS.at("ATTN"))) {
+            isolates.push_back(isol);
+        }
```

Comment:
This seem to impact the default partitioning (in REG), the tag needs to be stripped

Reply:
Will be gone with https://github.com/dmatveev/openvino/pull/22